components/cache/index.php

```diff
@@ -49,4 +50,37 @@
     echo $cache->get('test');
 });
 
+// NOTE: You will need to have redis running for this to work
+$app->get('/redis', function () {
+    $container = new Container;
+
+    $container['config'] = [
+        'cache.default'     => 'redis',
+        'cache.stores.redis' => [
+            'driver'     => 'redis',
+            'connection' => 'default'
+        ],
+        'cache.prefix' => 'illuminate_non_laravel',
+        'database.redis'    => [
+            'cluster' => false,
+            'default' => [
+                'host'     => '127.0.0.1',
+                'port'     => 6379,
+                'database' => 0,
+            ],
+        ]
+    ];
+
+    $container['redis'] = new Database($container['config']['database.redis']);
+
+    $cacheManager = new CacheManager($container);
+
+    $cache = $cacheManager->store();
+    // $cache = $cacheManager->store('redis');
```

Comment:
Sure thing, I removed it originally since its in the default example above, but I can just add it back in